### DISCUSSION TITLE
创建项目遇到ssl错误

### DISCUSSION THREAD

**Discussion by niceeee12138** [nonebot/discussions#425] [问答] [answered]
欢迎使用 NoneBot CLI!
[?] 你想要进行什么操作? 创建一个 NoneBot 项目.
[?] 选择一个要使用的模板: bootstap (初学者或用户)
正在加载适配器...
ModuleLoadFailed('Failed to get adapters list.', [ConnectError('[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1123)'), JSONDecodeError('Expecting value: line 1 column 1 (char 0)'), ConnectTimeout('timed out')])

搜索以后实在不知道怎么解决 求大佬看看

**Comment by yanyongyu** [comment, marked as answer]
你的nbcli版本是否是最新的？
